nestedtensor/csrc/nested_node.cpp

```diff
@@ -9,27 +9,38 @@ namespace nested_tensor {
 using namespace torch::jit;
 using namespace torch::autograd::utils;
 
-std::string _NestedNode___str__(const TensorNode& nested_node) {
+void pretty_print_tensor(std::stringstream& result, const std::string& python_tensor_string, const std::string& tabs) {
```

Comment:
"pretty print" is a bit of a misnomer because it's actually creating *the* str representation for this object and not the pretty version of a string representation of it (which is usually what I'd associate with pretty printing).